scripts/utils/delete_subscriptions.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+# Copyright 2024 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+import requests
+import sys
+
+requests.packages.urllib3.disable_warnings()
+
+__author__ = 'Md Arqum Farooqui'
+
+##############################################################
+# BEGIN of the configuration part (don't touch above this line ;)
+
+cb_endpoint = 'http://localhost:1026/v2/subscriptions'
+
+headers = {
+    'Accept': 'application/json',
+    'fiware-service': 'service',
+    'fiware-servicepath': '/subservice'
+}
+
+page_size = 500
+
+# WARNING! use this filter or you will remove *all* subscriptions
+filter = 'offset=0'
```

Comment:
Even with `offset=0`, all subscription would be removed :). Note that offset/limit are not actually filters, but a mechanism to get paginated results.

Let's keep the filter varible, just in case in the future gets implemented, but I'd suggest to use something like this in this point

```suggestion
# WARNING! use this filter or you will remove *all* subscriptions
# FIXME: note that filtering is not supported in the GET /v2/subscriptions. If at the it is get
# implemented (https://github.com/telefonicaid/fiware-orion/issues/1145 or additional issues),
# let's adjust this point of the script with a valid example filter
#filter = '&type=device'
```

Reply:
Yes, I intentionally put the offset value as the actual filter (q/type) is not working yet. 
But the suggested changes gives more clarity so I'll update the PR:)

Reply:
> Even with `offset=0`, all subscription would be removed :). Note that offset/limit are not actually filters, but a mechanism to get paginated results.
> 
> Let's keep the filter varible, just in case in the future gets implemented, but I'd suggest to use something like this in this point

filter variable cannot be commented as it make request bad. Please see below screenshot for reference:

![filter_variable](https://github.com/user-attachments/assets/bb8eee48-8ee0-4b4f-8fb3-d9b772b0ce7b)
 
So I can add below lines as well in the script:
```
# WARNING! use this filter or you will remove *all* subscriptions
# FIXME: note that filtering is not supported in the GET /v2/subscriptions. If at the it is get
# implemented (https://github.com/telefonicaid/fiware-orion/issues/1145 or additional issues),
# let's adjust this point of the script with a valid example filter
#filter = '&type=device'
# For now we can keep filter variable empty.
filter = ''
```

Reply:
Fine with it

Reply:
fixed in [01f8553](https://github.com/telefonicaid/fiware-orion/pull/4626/commits/01f855348599b3e69c8672ac89e28656b2e51a20)